pandas/core/groupby.py

```diff
@@ -191,62 +191,106 @@
     dtype: int64
     """)
 
-_numeric_operations_doc_template = """
-    Compute %(f)s of group values.
+_numeric_operations_full_names = dict(
+    sum='the sum',
+    prod='the product',
+    min='the minimum',
+    max='the maximum',
+    first='the first value',
+    last='the last value'
+)
+
+_numeric_operations_doc_template = dedent(
+    """
+    Compute %(full_name)s of each group.
 
-    For multiple groupings, the result index will be a
+    For multiple groupers, the result index will be a
     :class:`~pandas.MultiIndex`.
 
     Parameters
     ----------
     kwargs : dict
         Optional keyword arguments to pass to `%(f)s`.
 
+        `numeric_only` : bool
```

Comment:
Couldn't find an example in the docs where `kwargs` were listed out, so I've used the same convention used for listing out normal parameters.